android/src/toga_android/app.py

```diff
@@ -314,14 +315,18 @@ def set_current_window(self, window):
         pass
 
     ######################################################################
-    # Full screen control
+    # Presentation mode controls
     ######################################################################
 
-    def enter_full_screen(self, windows):
-        pass
+    def enter_presentation_mode(self, screen_window_dict):
+        for screen, window in screen_window_dict.items():
+            window._impl._before_presentation_mode_screen = window.screen
+            window.screen = screen
+            window._impl.set_window_state(WindowState.PRESENTATION)
 
-    def exit_full_screen(self, windows):
-        pass
+    def exit_presentation_mode(self):
+        # There is only a single window on android.
+        self.interface.main_window._impl.set_window_state(WindowState.NORMAL)
```

Comment:
I have removed any code dealing with screen changes since we do not currently support moving between screens. But Android does provide APIs to set a window on a particular screen. In the future, if required, I'll create a PR for supporting the same. But for now, I'll leave it from this PR.